packages/web3-eth-contract/test/unit/contract.test.ts

```diff
@@ -72,6 +72,18 @@ describe('Contract', () => {
 
 			expect(contract).toBeInstanceOf(Contract);
 		});
+
+		it('should set the provider upon instantiation', () => {
+			const provider = 'http://127.0.0.1:8545';
```

Comment:
We are better to avoid string constants.

```suggestion
			const provider = getSystemTestProvider();
```
(you would need to import `getSystemTestProvider` from the file `system_test_utils`)

Reply:
Addressed by [this](https://github.com/web3/web3.js/pull/5669/commits/16b6bee9ad1efc53db6e4a1427be2b2e78972d78) commit